hotkey.js

```diff
@@ -14,10 +14,50 @@ function ignoreHotkeysIn(target) {
   )
 }
 
+function isElementIsIgnoredInRange(parent, node) {
+  if (node.tagName === 'BODY') return null
+  if (parent === node) return null
+  if (node.hasAttribute('data-keyux-ignore-hotkeys')) return node
+
+  return isElementIsIgnoredInRange(parent, node.parentNode)
```

Comment:
Will `Element#contains` will faster and smaller?

Reply:
@ai We want to find first "ignored" element between activeElement and element with "aria-keyshortcuts".
We may have 2 cases:

```
        <ul>
          <li
            data-keyux-ignore-hotkeys
          >
            <button aria-keyshortcuts="v">First button</button>
          </li>
          <li>
            <button aria-keyshortcuts="v">Second button</button>
          </li>
        </ul>
```

in first case with li it's valid. But we don't have any ignored elements between last li and body. if we will use activeElement.contains it will return true and we won't call "Second button"